pandas/io/formats/style.py

```diff
@@ -359,7 +359,8 @@ def concat(self, other: Styler) -> Styler:
         When ``other`` has fewer index levels than the original Styler it is possible
         to extend the index in ``other``, with placeholder levels.
 
-        >>> df = DataFrame([[1], [2]], index=pd.MultiIndex.from_product([[0], [1, 2]]))
+        >>> df = pd.DataFrame([[1], [2]],
+        ...             index=pd.MultiIndex.from_product([[0], [1, 2]]))
```

Comment:
the indentation looks a bit odd here, if you just run `validate_docstrings` on just this function, I think you might get an EX03 error about that? If so, could you better align it please? I think something like
```
        >>> df = pd.DataFrame([[1], [2]],
        ...                   index=pd.MultiIndex.from_product([[0], [1, 2]]))
```
should do